go/enclave/rollupchain/rollup_chain.go

```diff
@@ -502,15 +515,16 @@ func (rc *RollupChain) SubmitL1Block(block types.Block, isLatest bool) (*common.
 	}
 
 	rc.logger.Trace(fmt.Sprintf("Update state: b_%d", common.ShortHash(block.Hash())))
-	blockState := rc.updateState(&block)
+	// TODO - Handle error properly.
+	blockState, _ := rc.updateState(&block)
 	if blockState == nil {
 		// not an error state, we ingested a block but no rollup head found
 		return rc.noBlockStateBlockSubmissionResponse(&block), nil
 	}
 
 	logs := []*types.Log{}
-	fetchedLogs, found := rc.storage.FetchLogs(block.Hash())
-	if found {
+	fetchedLogs, err := rc.storage.FetchLogs(block.Hash())
+	if err == nil {
 		logs = fetchedLogs
 	} else {
 		rc.logger.Error("Could not retrieve logs for stored block state. Returning no logs")
```

Comment:
Can the error be included in the log message ?

Reply:
Yep